vmdb/app/assets/javascripts/dynatree_replacement.js

```diff
@@ -0,0 +1,164 @@
+var dynatreeReplacement = {
```

Comment:
Let's call this file miq_dynatree_replacement.js and the object miqDynatreeReplacement.